apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/template/index.js

```diff
@@ -19,7 +19,7 @@ if ( 'wp_template_part' !== fullSiteEditing.editorPostType ) {
 		category: 'layout',
 		attributes: { templateId: { type: 'number' } },
 		supports: {
-			align: [ 'wide', 'full' ],
+			align: [ 'full' ],
```

Comment:
I think we can remove this support and use `getEditWrapperProps` to force always a full align, while ensuring the user is not able to change the alignment. 

Since you're AFK tomorrow, I'll commit a change based on that (hope you don't mind!).

Reply:
Done in 2a10fb6. This also removes the necessity of hiding the block controls with CSS, solving the issue with the top toolbar mode enabled :)